functions/node8/package.json

```diff
@@ -19,7 +19,9 @@
     "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples/issues"
   },
   "dependencies": {
-    "ava": "^0.25.0"
+    "@google-cloud/nodejs-repo-tools": "^2.3.3",
```

Comment:
`repo-tools` and `ava` should be dev dependencies.

Reply:
Done.